tools/cldr-code/src/main/resources/org/unicode/cldr/tool/modify_config.txt

```diff
@@ -1,196 +1,305 @@
+locale=tk ; action=add ; new_path=//ldml/dates/calendars/calendar[@type="islamic"]/months/monthContext[@type="stand-alone"]/monthWidth[@type="wide"]/month[@type="5"][@draft="contributed"] ; new_value=Dört tirkeşik 3
+locale=tk ; action=add ; new_path=//ldml/dates/calendars/calendar[@type="islamic"]/months/monthContext[@type="stand-alone"]/monthWidth[@type="wide"]/month[@type="6"][@draft="contributed"] ; new_value=Dört tirkeşik 4
+locale=tk ; action=add ; new_path=//ldml/dates/calendars/calendar[@type="islamic"]/months/monthContext[@type="stand-alone"]/monthWidth[@type="wide"]/month[@type="7"][@draft="contributed"] ; new_value=Rejep
+locale=tk ; action=add ; new_path=//ldml/dates/calendars/calendar[@type="islamic"]/months/monthContext[@type="stand-alone"]/monthWidth[@type="wide"]/month[@type="8"][@draft="contributed"] ; new_value=Meret
+locale=tk ; action=add ; new_path=//ldml/dates/calendars/calendar[@type="islamic"]/months/monthContext[@type="stand-alone"]/monthWidth[@type="wide"]/month[@type="9"][@draft="contributed"] ; new_value=Oraza
+locale=tk ; action=add ; new_path=//ldml/dates/calendars/calendar[@type="islamic"]/months/monthContext[@type="stand-alone"]/monthWidth[@type="wide"]/month[@type="10"][@draft="contributed"] ; new_value=Baýram
+locale=tk ; action=add ; new_path=//ldml/dates/calendars/calendar[@type="islamic"]/months/monthContext[@type="stand-alone"]/monthWidth[@type="wide"]/month[@type="11"][@draft="contributed"] ; new_value=Boş aý
+locale=tk ; action=add ; new_path=//ldml/dates/calendars/calendar[@type="islamic"]/months/monthContext[@type="stand-alone"]/monthWidth[@type="wide"]/month[@type="12"][@draft="contributed"] ; new_value=Gurban
+locale=tk ; action=add ; new_path=//ldml/dates/calendars/calendar[@type="islamic"]/months/monthContext[@type="format"]/monthWidth[@type="abbreviated"]/month[@type="1"][@draft="contributed"] ; new_value=Aşyr
+locale=tk ; action=add ; new_path=//ldml/dates/calendars/calendar[@type="islamic"]/months/monthContext[@type="format"]/monthWidth[@type="abbreviated"]/month[@type="2"][@draft="contributed"] ; new_value=Sap.
+locale=tk ; action=add ; new_path=//ldml/dates/calendars/calendar[@type="islamic"]/months/monthContext[@type="format"]/monthWidth[@type="abbreviated"]/month[@type="3"][@draft="contributed"] ; new_value=4 tir 1
+locale=tk ; action=add ; new_path=//ldml/dates/calendars/calendar[@type="islamic"]/months/monthContext[@type="format"]/monthWidth[@type="abbreviated"]/month[@type="4"][@draft="contributed"] ; new_value=4 tir 2
+locale=tk ; action=add ; new_path=//ldml/dates/calendars/calendar[@type="islamic"]/months/monthContext[@type="format"]/monthWidth[@type="abbreviated"]/month[@type="5"][@draft="contributed"] ; new_value=4 tir 3
+locale=tk ; action=add ; new_path=//ldml/dates/calendars/calendar[@type="islamic"]/months/monthContext[@type="format"]/monthWidth[@type="abbreviated"]/month[@type="6"][@draft="contributed"] ; new_value=4 tir 4
```

Comment:
Should this file really be checked in? Maybe it should be .gitignored

Reply:
I think it is actually useful to have a store of these as examples.

Reply:
I haven't used this tool.  However, it does seem (looking at history) a little random to check in the file each time with the latest result.  If we want to have an example modify_config.txt or set of them for different purposes, that could be good.

Might be better to take a filesystem path rather than an in-source-tree path, at least as an option.  That would be what I'd expect from an occasional tool.